frontend/e2e/fixtures/test-fixtures.ts

```
@@ -0,0 +1,174 @@
+   * Uses saved storage state for authentication
+   */
+  authenticatedPage: async ({ page }, use) => {
+    // The storage state is automatically loaded from config
+    await use(page)
+  },
+
+  /**
+   * Test prefix for unique naming
+   * Helps avoid conflicts between test runs
+   */
+  testPrefix: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)